objects.md

```diff
@@ -0,0 +1,118 @@
+# About
+
+This repo needs to represent the following information:
+
+1. Adding/removing versions to channels
```

Comment:
"Adding releases to channels"?  We never remove them from channels to avoid stranding users who need off ramps.

Reply:
agree that I don't currently know a reason to 'remove'